docs/_documentations/importing-existing-projects.md

```diff
@@ -176,7 +176,7 @@ If you have a `Dockerfile`, it must meet the following requirements:
     - For example, if you're using port 3000, the `Dockerfile` needs to include `EXPOSE 3000`.
 - Ensure that the application is located in the `/app` directory within the Docker container.
 - Ensure that the `Dockerfile` sets the working directory to `/app`:
-    - `WORKDIR "/app"`
+    - `WORKDIR "/app"`.
```

Comment:
How about...`Ensure that the `Dockerfile` sets the working directory to `/app` with `WORKDIR "/app"`.`